ssc/cmod_pvsamv1.cpp

```diff
@@ -2725,11 +2772,12 @@ void cm_pvsamv1::exec()
                 //Calculate DNI clearness index (time step basis)
                 double dni_clearness_index = PVSystem->p_DNIIndex[0][idx];
                 //Calculate Clipping Potential ((P_dc,dryclean - P_ac,0) / P_ac,0) (time step basis)
-                sharedInverter->calculateACPower(dcPower_kW_csky, dcVoltagePerMppt[0], Irradiance->weatherRecord.tdry, as_boolean("enable_subhourly_clipping")); //DC batteries not allowed with multiple MPPT, so can just use MPPT 1's voltage
+                //sharedInverter->calculateACPower(dcPower_kW_csky, dcVoltagePerMppt[0], Irradiance->weatherRecord.tdry, as_boolean("enable_subhourly_clipping")); //DC batteries not allowed with multiple MPPT, so can just use MPPT 1's voltage
                 //double clip_pot = (dcPower_kW_csky - sharedInverter->powerAC_kW_clipping) / sharedInverter->powerAC_kW_clipping;
                 double clip_pot = (dcPower_kW_csky - paco ) / (paco);
                 
                 PVSystem->p_ClippingPotential[idx] = clip_pot;
+                //PVSystem->p_DNIIndex[idx] = dni_clearness_index;
```

Comment:
When the PR is set to your liking, please remove commented out lines to help keep code a bit cleaner